src/common/components/Table/Table.tsx

```diff
@@ -78,6 +81,21 @@ const Table = ({
     }
   }
 
+  const renderSearhKeyDiv = (param: string, text: JSX.Element | string) => {
```

Comment:
`dangerouslySetInnerHTML` is named so because if user will put a javascript tag with alert into nft description it will bother other users, and in worst case user can put js tag with js code to steal other users secrets.

https://stackoverflow.com/questions/29652862/highlight-text-using-reactjs
Here is example how to do it safely, please update code in same way

I don't think standalone library is needed here. Are we unable to get text without library? Does it have to be JSX.Element, can't we get text from data?

Reply:
Sometimes we are getting dom element as well as text from the table custom column, so we need to change it into the text and need to replace it with highlighted text and return dom element again.